script/gather_stake_accounts.rb

```diff
@@ -24,6 +24,20 @@
 
   TotalRewardsUpdateService.new(p.payload[:network], p.payload[:stake_accounts_active]).call
 
+  next unless network == 'mainnet'
```

Comment:
This would skip the logging that we do at the end of this script. We only want to skip the GatherExplorerStakeAccountsService service.